jobs/mozaggregator2bq/bin/load_bq

```diff
@@ -4,8 +4,8 @@
 set -x
 
 REPLACE=${REPLACE:-false}
-PROJECT="mozaggregator2bq"
-DATASET="telemetry"
+PROJECT="moz-fx-data-shared-prod"
```

Comment:
that is not the same project that you use in line 37? `moz-fx-data-prod-external-data`

Reply:
The GCP project is `moz-fx-data-shared-prod` and the bucket (that belongs to that project) where the data gets stored is `moz-fx-data-prod-external-data`